### ISSUE TITLE
Neovim's virtualtext for 

### ISSUE THREAD

**Issue by blueyed** [davidhalter/jedi-vim#891]
We could investigate using Neovim's virtualtext support for call signatures (https://github.com/davidhalter/jedi-vim/issues/890#issuecomment-444284185), which could be added to the end of the current line.

(https://github.com/davidhalter/jedi-vim/issues/890#issuecomment-444284185 reminded me of that)

**Comment by blayz3r** [comment]
Does the new Vim popup change anything? Syntax highlighting signatures maybe?

**Comment by blueyed** [comment]
Yeah, that would likely be even better to use.  Neovim has it also (had it before, and Vim uses a different API again...).

**Comment by davidhalter** [comment]
I'm open to all those changes, but I'm still using old VIM, so feel free to change this.

**Comment by blueyed** [comment]
Started playing with using Neovim's floating windows.
Based on https://github.com/davidhalter/jedi-vim/pull/652.

**Comment by blueyed** [comment]
> Syntax highlighting signatures maybe?

It still uses the existing concealing (but in a separate window, not messing with the current buffer anymore, which might interfere with indenting etc).

**Comment by davidhalter** [comment]
We can do the same for VIM, using `textprop`, but we need at least patch 8.2.0286.

**Comment by blueyed** [comment]
Note: textprop is used for usages already: https://github.com/blueyed/jedi-vim/blob/6ca603ffdf989cfc7426485252a77529951ea47e/pythonx/jedi_vim.py#L658-L669
